lib/sourcebit.js

```diff
@@ -1,22 +1,27 @@
+const { cloneDeep } = require('lodash');
 const debug = require('debug');
+const { diff: generateDiff } = require('deep-diff');
 const fs = require('fs');
 const mkdirp = require('mkdirp');
 const ora = require('ora');
 const path = require('path');
-const { cloneDeep } = require('lodash');
+const util = require('util');
+
 const { writeFrontmatterMarkdown, writeJSON, writeYAML } = require('./file-writers');
 
-const FILE_WRITERS = {
+const fileWriters = {
     'frontmatter-md': writeFrontmatterMarkdown,
     json: writeJSON,
     yml: writeYAML
 };
```

Comment:
@eduardoboucas you can use this battle-tested utility `stringifyDataByFilePath` to write all sort of files:
https://github.com/stackbithq/utils/blob/d1222b63ede765fb895668b641cb3f44c238d6ea/src/index.js#L373-L394
It takes into account several quirks that will prevent the app from crashing and store files safely.